realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -61,6 +61,7 @@
 
     private static final Object DEFAULT_MODULE;
     private static final RealmProxyMediator DEFAULT_MODULE_MEDIATOR;
+    private static Boolean rxJavaAvailable;
```

Comment:
`rxJavaAvailable` can be `null`/`false`/`true` . `null` means it is not checked yet.